profile/README.md

```diff
@@ -29,7 +29,7 @@
        <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
    </a>&nbsp;&nbsp;&nbsp;
    <a href="https://twitter.com/revancedapp">
-       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032018-6da37214-7474-4641-a1da-7af7db3a31cd.png" />
+       <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
```

Comment:
Please add a light and dark-themed icon. On a dark background, this logo is invisible.